code/game/sound.dm

```diff
@@ -56,27 +66,25 @@
 			else //space
 				pressure_factor = 0
 
-			var/distance = get_dist(T, turf_source)
 			if(distance <= 1)
 				pressure_factor = max(pressure_factor, 0.15) //touching the source of the sound
 
 			S.volume *= pressure_factor
 			//End Atmosphere affecting sound
 
-			if(S.volume <= 0)
-				return //No sound
-
-		// 3D sounds, the technology is here!
-		if (surround)
-			var/dx = turf_source.x - T.x // Hearing from the right/left
-			S.x = round(max(-SURROUND_CAP, min(SURROUND_CAP, dx)), 1)
-
-			var/dz = turf_source.y - T.y // Hearing from infront/behind
-			S.z = round(max(-SURROUND_CAP, min(SURROUND_CAP, dz)), 1)
+		if(S.volume <= 0)
+			return //No sound
 
+		var/dx = turf_source.x - T.x // Hearing from the right/left
+		S.x = dx
+		var/dz = turf_source.y - T.y // Hearing from infront/behind
+		S.z = dz
 		// The y value is for above your head, but there is no ceiling in 2d spessmens.
 		S.y = 1
-		S.falloff = falloff || FALLOFF_SOUNDS
+		S.falloff = (falloff ? falloff : FALLOFF_SOUNDS)
```

Comment:
Random fact; Goon doesn't use arbitrary distance limits at all; they depend entirely on `falloff` to govern how loud/soft a sound is by the time it reaches someone.

It generates a really cool effect, but it also means you can just about hear everything that goes on, in station, if you have your sound turned up (RIP stealth killing someone in deep south-east maintenance because an assistant who just arrived hear you).